src/data/sql-quiz.ts

```diff
@@ -1099,6 +1099,32 @@ const sqlQuiz = [
     Explanation:
       "The key difference between HAVING and WHERE is when the filtering occurs: HAVING works after aggregation, while WHERE works before aggregation.",
     Link: "https://www.freecodecamp.org/news/the-sql-having-clause-explained-with-example-syntax/"
+  },
+  {
+    Question:
+      "What is the difference between 'CHAR' and 'VARCHAR' data types in SQL?",
+    Answer:
+      "'CHAR' holds a fixed string length. 'VARCHAR' holds a variable string length",
+    Distractor1:
+      "'CHAR' is a numeric data type used for storing integers, while 'VARCHAR' is a text data type for storing strings.",
+    Distractor2: "'CHAR' is a case-insensitive data type, while 'VARCHAR' is case-sensitive.",
+    Distractor3: "'CHAR' is used for storing fixed-length data, while 'VARCHAR' is used for storing variable-length data.",
+    Explanation:
+      "CHAR holds a fixed string length because it allocates a set amount of storage for the fixed size specified in parenthesis, VARCHAR holds a variable string length by the maximum size specified in parenthesis.",
```

Comment:
```suggestion
      "CHAR holds a fixed string length because it allocates a set amount of storage for the fixed size specified in parenthesis. VARCHAR holds a variable string length by the maximum size specified in parenthesis.",
```